README.md

```diff
@@ -1,2 +1,19 @@
 # Greetings-Microverse
-Hello Microverse
+Microverse first project.
+
+## Version
+Version 1.0.0
+
+## Contributors
+
+**Author**
+Name : Adoyo Alphonce;
+Email : apondimama5@gmail.com;
+Github : https://github.com/tingamapuro04
+
+
+## Description
+This is a small project generally looking at github workflow where i was setting a linters.yml file, adding a .gitignore and then created a heading and added a color to it.
+
+## Copyright
+© Adoyo A Mobutu
```

Comment:
- [ ] [OPTIONAL] kindly you have to make change requested by the previous review. you're almost there but follow this [microverse Readme](https://github.com/microverseinc/readme-template/blob/master/README.md)